aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_configurator.py

```diff
@@ -319,10 +319,19 @@ def _customize_exporter(span_exporter: SpanExporter, resource: Resource) -> Span
             traces_endpoint = os.environ.get(AWS_XRAY_DAEMON_ADDRESS_CONFIG, "127.0.0.1:2000")
             span_exporter = OTLPUdpSpanExporter(endpoint=traces_endpoint)
 
-    if isinstance(span_exporter, OTLPSpanExporter) and is_xray_otlp_endpoint(
-        os.environ.get(OTEL_EXPORTER_OTLP_TRACES_ENDPOINT)
-    ):
-        span_exporter = OTLPAwsSpanExporter(endpoint=os.getenv(OTEL_EXPORTER_OTLP_TRACES_ENDPOINT))
+    if is_xray_otlp_endpoint(os.environ.get(OTEL_EXPORTER_OTLP_TRACES_ENDPOINT)):
+        # TODO: Change this url once doc writer has added a section for using SigV4 without collector
+        _logger.info(
+            "Detected using AWS OTLP XRay Endpoint. Please ensure you've followed the documentation to properly setup your environment to export traces to XRay. See: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-OpenTelemetry-Sections.html"
+        )
```

Comment:
I don't think we need this log statement or can simply be "Detected using AWS OTLP XRay Endpoint". Users who want to send directly to the OTLP endpoint should discover this feature from the docs and should set up accordingly, then come to code. IMO if logs can't give specific instructions to solve a problem on the spot, they shouldn't point to a big doc for pre-configurations.